frontend/src/js/store/organizationSlice/thunks.ts

```
@@ -310,7 +310,11 @@ export const getBillingPreview = createAsyncThunk(`${sliceName}/getBillingPrevie
 );
 
 export const getCurrentSubscription = createAsyncThunk(`${sliceName}/getCurrentSubscription`, () =>
-  Api.get(`${tenantadmApiUrlv2}/billing/subscription`).then(res => res.data)
+  Api.get(`${tenantadmApiUrlv2}/billing/subscription`).then(res => res.data).catch(e=> {
+    if (e.response && e.response.status === 404){
+      throw new Error('Request failed with status code 404')
+    }
+  })
 );
```

considering that the error gets eaten up otherwise, should this not forward all others error instead or use the `commonErrorHandler`? otherwise I would assume the `getUserSubscription` above would return without user feedback in case they have a regular subscription and would expect to be able to adjust it.